diff --git a/bin.cjs b/bin.cjs
index 327fdfde1aac8ddf7a0db6dc357796757be8986a..17007ebb357da91b8ad9d4c9192e45bcd1647575 100755
--- a/bin.cjs
+++ b/bin.cjs
@@ -19575,6 +19575,7 @@ WHERE
             for (const checks of tableChecks) {
               let checkValue = checks.constraint_definition;
               const constraintName = checks.constraint_name;
+              if (!checkValue) continue;
               checkValue = checkValue.replace(/^CHECK\s*\(\(/, "").replace(/\)\)\s*$/, "");
               checkConstraints[constraintName] = {
                 name: constraintName,
